Gems/LmbrCentral/Code/Tests/Builders/CopyDependencyBuilderTest.cpp

```diff
@@ -896,7 +894,7 @@ namespace UnitTest
         AssetBuilderSDK::CreateJobsResponse response;
 
         request.m_sourceFile = "Tests/Xmls/XmlExampleWithoutVersion.xml";
-        request.m_watchFolder = "@root@/../Gems/LmbrCentral/Code/";
+        request.m_watchFolder = "@projectproductassets@/../Gems/LmbrCentral/Code/";
```

Comment:
This should be `@engroot@/Gems/LmbrCentral/Code/`

Reply:
Will change.